security/security-auth-kylo-groups/src/main/java/com/thinkbiganalytics/auth/groups/KyloGroupsLoginModule.java

```diff
@@ -119,7 +119,7 @@ protected boolean doLogin() throws Exception {
         try {
             groups = retrieveGroups(userConfig);
         } catch (Exception e) {
-            log.error("An error occurred while getting Kylo Groups username: {}", username);
+            log.error("An error occurred while getting Kylo Groups username: {}", username, e);
             throw new RuntimeException("An error occurred while getting Kylo Groups", e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)